.script/utils/schemas/WorkbooksMetadataSchema.json

```diff
@@ -0,0 +1,75 @@
+{
+    "type": "array",
+    "items": [
+        {
+            "type": "object",
+            "properties": {
+                "workbookKey": {
```

Comment:
Is there an option to assert that all the keys are distinct? To make sure no one added a workbook with existing key?

Reply:
From what I understand it's [not possible](https://stackoverflow.com/questions/24763759/make-sure-item-property-in-array-is-unique-in-json-schema) using a json schema. However we can create a custom validation for this (with ts). I listed the validations you mentioned here and I will create a separate validator job specifically for workbooks.